addons/website_slides/models/slide_channel.py

```diff
@@ -402,6 +415,8 @@ def write(self, vals):
         # If description_short wasn't manually modified, there is an implicit link between this field and description.
         if vals.get('description') and not vals.get('description_short') and self.description == self.description_short:
             vals['description_short'] = vals.get('description')
+        if not self.image_1920 and not vals.get('image_1920'):
```

Comment:
if not vals.get('image_1920', self.image_1920) is equivalent I think

Reply:
Please avoid storing placeholder images in database, as it will pollute the attachment table with tons of duplicate of the default image, and it will later be impossible to know which records possesses the default image and which have an actual image set.

It is preferable to handle "empty image" in views, and displaying the placeholder there if necessary.

There are also perf reasons, see https://github.com/odoo/odoo/pull/36093

Reply:
As the hunt against the 'there is no image so we put one by default' is the new top priority, I guess this needs to be checked with the PO to remove that.

Reply:
Sorry if it wasn't clear, but just to confirm, your last example is indeed what I